### ISSUE TITLE
Are the changes made in 1.11 reflected in the docs?

### ISSUE THREAD

**Issue by thisisthekap** [MicrosoftDocs/appcenter-docs#400]
Are the changes of 1.11 reflected in the docs? Especially this one:

>[Fix] Fix PushNotificationReceived callback not triggered on notification tapped or received in foreground when a UNUserNotificationCenterDelegate is set. If you have implemented this delegate please remove any call to the Push.DidReceiveRemoteNotification method as it's now handled by the new User Notification Center Delegate Forwarder.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b6415607-5d22-0249-2537-1d7887b46f16
* Version Independent ID: 32fafbc0-515e-043f-3481-6f7c3fe2cc91
* Content: [App Center Push for Xamarin.iOS Apps - Visual Studio App Center](https://docs.microsoft.com/en-us/appcenter/sdk/push/xamarin-ios)
* Content Source: [docs/sdk/push/xamarin-ios.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/sdk/push/xamarin-ios.md)
* Service: **vs-appcenter**
* GitHub Login: @elamalani
* Microsoft Alias: **emalani**

**Comment by jwargo** [comment]
@thisisthekap I'll have the PM take a look.

**Comment by jaeklim** [comment]
@thisisthekap We've updated the doc based on what we've changed [here](https://github.com/MicrosoftDocs/appcenter-docs/commit/bf5bb8941d0c601a0d22e322b4cee7bb11cbcae3#diff-f2140d6a18cf520c5199b6107d87bef7).
The instruction that was mentioned in the CHANGELOG is still required for users who have `UNUserNotificationCenterDelegate` in apps and call `Push.DidReceiveRemoteNotification` from there. The doc doesn't have any instructions for SDK update, sorry for the confusions. I'll file a bug to improve the doc with more info.
